airflow/providers/google/cloud/example_dags/example_dataflow.py

```diff
@@ -108,9 +114,17 @@
     )
     # [END howto_operator_start_python_job]
 
+    # [START howto_operator_start_local_python_job]
+    py_to_local = GCSToLocalOperator(
```

Comment:
This should not be found in the guide. This operator is part of DAG for testing only. We want to test if a local file and a GCS file can be run.

Reply:
[Done here](https://github.com/apache/airflow/pull/8809/commits/25d8d082f2502cd64ab7c88e8988799d46fe040a)